src/test/java/com/qdesrame/openapi/test/BackwardCompatibilityTest.java

```diff
@@ -37,14 +39,24 @@ public void testApiChangedOperationAdded() {
   public void testApiChangedOperationMissing() {
     assertOpenApiBackwardIncompatible(OPENAPI_DOC3, OPENAPI_DOC2);
   }
+  
+  @Test
+  public void testApiReadWriteOnlyPropertiesChanged() {
+    assertOpenApiBackwardCompatible(OPENAPI_DOC1, OPENAPI_DOC5, true);
+  }
 
   @Test
-  public void testApiOperationChanged() {
```

Comment:
This test was failing and upon inspection of the test files it wasn't testing anything to do with operations. A default was missing in the second test file which is why it's now failing.

Reply:
wonder what they were trying to test here...